core.py

```diff
@@ -15,10 +15,11 @@ class ZulipController:
     the application.
     """
 
-    def __init__(self, config_file: str) -> None:
+    def __init__(self, config_file: str, theme:str) -> None:
```

Comment:
`theme: str='default'`
we don't want to make it compulsory to add a theme while running the program.

Reply:
The default is mentioned in arg parser.
Value of theme is 'default' initially.